man/systemd.network.xml

```diff
@@ -2257,6 +2257,53 @@
       </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[TrafficControlQueueingDiscipline] Section Options</title>
+    <para>The <literal>[TrafficControlQueueingDiscipline]</literal> section manages the Traffic control. It can be used
+    to configure the kernel packet scheduler and simulate packet delay and loss for UDP or TCP applications,
+    or limit the bandwidth usage of a particular service to simulate internet connections.</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Parent=</varname></term>
+        <listitem>
+          <para>Specifies Queueing Discipline (qdisc). Takes one of <literal>root</literal> or <literal>clsact</literal>.
+          Defaults to <literal>root</literal>.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelaySec=</varname></term>
+        <listitem>
+          <para>Specifies the network fixed amount of delay to all packets going out of the interface. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayJitterSec=</varname></term>
+        <listitem>
+          <para>Specifies the random delay to be added to <varname>NetworkDelaySec=</varname>. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetworkDelayLimitPacket=</varname></term>
+        <listitem>
+          <para>Specifies the fifo limit (packets). An unsigned integer ranges 0 to 4294967294. Defaults to 100.</para>
```

Comment:
"Specifies the maximum number of packets the qdisc may hold queued at a time." (from the man page)

s/ranges/in the range/

Reply:
Then what they suggest

Reply:
I'm suggesting to replace the description with the text I quoted from the man page

Reply:
I think that is completely ok.

Reply:
thanks